File: 0168-excel-sheet-column-title/0168-excel-sheet-column-title.py
class Solution:
    def convertToTitle(self, columnNumber: int) -> str:
        result = list()
        while columnNumber:
            idx = (columnNumber - 1) % 26
            result.append(chr(idx + ord('A')))
            columnNumber = (columnNumber - 1) // 26
        return ''.join(result[::-1])

File: 0168-excel-sheet-column-title/0168-excel-sheet-column-title.cpp
class Solution {
public:
    string convertToTitle(int columnNumber) {
        string result = "";
        while (columnNumber > 0) {
            int idx = (columnNumber - 1) % 26;
            result = (char)('A' + idx) + result;
            columnNumber = (columnNumber - 1) / 26;
        }
        return result;
    }
};